snuba/migrations/snuba_migrations/transactions/0005_transactions_add_http_fields.py

```diff
@@ -0,0 +1,83 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, LowCardinality, Nullable, String
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+
+
+class Migration(migration.MultiStepMigration):
+    """
+    Adds the http columns populated from the Request interface that is missing from transactions.
+    """
+
+    blocking = True
```

Comment:
Is there a reason it needs to be marked true? This is intended for data migrations / things that may take some time.

Reply:
I guess related to this, were you planning to provide a migration that actually populates data into these columns from somewhere?

Reply:
I'll remove it then.